pennylane_qiskit/basic_aer.py

```diff
@@ -58,10 +58,56 @@ def __init__(self, wires, shots=1024, backend="qasm_simulator", **kwargs):
             raise RuntimeError(
-                f"The devices in the PennyLane Qiskit plugin are currently only compatible "
-                f"with versions of Qiskit below 0.46. You have version {qiskit.__version__} "
-                f"installed. Please downgrade Qiskit to use the devices. The devices will be "
-                f"updated in the coming weeks to be compatible with Qiskit 1.0!"
+                f"Qiskit has discontinued the BasicAer device, so it can only be used in"
+                f"versions of Qiskit below 0.46. You have version {qiskit.__version__} "
+                f"installed. For a Python simulator, use the 'qiskit.basicsim' device "
+                f"instead. Alternatively, you can downgrade Qiskit to use the "
+                f"'qiskit.basicaer' device."
             )
 
         super().__init__(wires, provider=qiskit.BasicAer, backend=backend, shots=shots, **kwargs)
+
+
+class BasicSimulatorDevice(QiskitDevice):
+    """A PennyLane device for the native Python Qiskit simulator BasicSimulator.
+
+    Please see the `Qiskit documentations <https://qiskit.org/documentation/>`_
+    further information on the backend options and transpile options.
+
+    A range of :code:`backend_options` that will be passed to the simulator and
+    a range of transpile options can be given as kwargs.
+
+    For more information on backends, please visit the
+    `BasicProvider documentation <https://docs.quantum.ibm.com/api/qiskit/providers_basic_provider>`_.
+
+    Args:
+        wires (int or Iterable[Number, str]]): Number of subsystems represented by the device,
+            or iterable that contains unique labels for the subsystems as numbers (i.e., ``[-1, 0, 2]``)
+            or strings (``['aux_wire', 'q1', 'q2']``).
+        backend (str): the desired backend
+        shots (int or None): number of circuit evaluations/random samples used
+            to estimate expectation values and variances of observables. For statevector backends,
+            setting to ``None`` results in computing statistics like expectation values and variances analytically.
+
+    Keyword Args:
+        name (str): The name of the circuit. Default ``'circuit'``.
+        compile_backend (BaseBackend): The backend used for compilation. If you wish
+            to simulate a device compliant circuit, you can specify a backend here.
+    """
+
+    short_name = "qiskit.basicsim"
+
+    def __init__(self, wires, shots=1024, backend="basic_simulator", **kwargs):
+
+        min_version = Version("1.0.0")
+
+        if Version(qiskit.__version__) < min_version:
+            raise RuntimeError(
+                f"The 'qiskit.simulator' device is not compatible with version of Qiskit prior "
+                f"to 1.0. You have version {qiskit.__version__} installed. For a Python simulator, "
+                f"use the 'qiskit.basicaer' device instead. Alternatively, upgrade Qiskit "
+                f"to use the 'qiskit.basicsim' device."
```

Comment:
Would be nice for us to give a link for those upgrade and downgrade instructions.

Reply:
added a link to their page about installing/upgrading to 1.0